Symfony/CS/Fixer/Symfony/SingleQuoteFixer.php

```diff
@@ -34,13 +34,14 @@ public function fix(\SplFileInfo $file, $content)
             $content = $token->getContent();
             if (
                 '"' === $content[0] &&
-                false === strpos($content, "'") &&
-                // regex: odd number of backslashes, not followed by double quote
-                !preg_match('/(?<!\\\\)(?:\\\\{2})*\\\\(?!["\\\\])/', $content, $m)
+                // regex: odd number of backslashes, not followed by dollar or double quote
+                !preg_match('/(?<!\\\\)(?:\\\\{2})*\\\\(?![("|\$)\\\\])/', $content, $m)
```

Comment:
> `[("|\$)\\\\]`

This looks wrong to me. You are mixing character classes (`[...]`) with alternations (`(...|...)`).